tracker/gioui/numericupdown.go

```diff
@@ -74,11 +80,22 @@ func NumericUpDown(th *material.Theme, number *NumberInput, tooltip string) Nume
 		Tooltip:         Tooltip(th, tooltip),
 		Width:           unit.Dp(70),
 		Height:          unit.Dp(20),
+		Padding:         unit.Dp(padding),
 		shaper:          *th.Shaper,
 	}
 }
 
 func (s *NumericUpDownStyle) Layout(gtx C) D {
+	if s.Hidden {
```

Comment:
again, I'm wondering if we need Hidden, or can just skip calling Layout altogether? You can make "empty" or "null" widget just by making a function something like:

```
func EmptyWidget(gtx C) D {
   return D{}
}
```

that you can pass to any function that expects a widget and you don't want one.

Reply:
yes, see above